ci/indy-pool.dockerfile

```diff
@@ -12,32 +12,38 @@ RUN apt-get update -y && apt-get install -y \
 	python3-nacl \
 	apt-transport-https \
 	ca-certificates \
+	software-properties-common \
 	supervisor
 
 RUN pip3 install -U \
-	pip==9.0.3 \
-	setuptools
+	"pip~=9.0" \
+	"setuptools~=50.0"
 
-RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys CE7709D068DB5E88 || \
+RUN add-apt-repository "deb http://us.archive.ubuntu.com/ubuntu xenial main universe" && \
 	apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys CE7709D068DB5E88
 ARG indy_stream=master
-RUN echo "deb https://repo.sovrin.org/deb xenial $indy_stream" >> /etc/apt/sources.list
+RUN add-apt-repository "deb https://repo.sovrin.org/deb xenial ${indy_stream}" && \
+	add-apt-repository "deb https://repo.sovrin.org/sdk/deb xenial stable"
 
 RUN useradd -ms /bin/bash -u $uid indy
 
-ARG indy_plenum_ver=1.12.1~dev989
-ARG indy_node_ver=1.12.1~dev1172
-ARG python3_indy_crypto_ver=0.4.5
-ARG indy_crypto_ver=0.4.5
-ARG python3_pyzmq_ver=18.1.0
+ARG indy_plenum_ver=1.13.0.dev1032
+ARG indy_node_ver=1.13.0.dev1221
 
 RUN apt-get update -y && apt-get install -y \
-        python3-pyzmq=${python3_pyzmq_ver} \
-        indy-plenum=${indy_plenum_ver} \
-        indy-node=${indy_node_ver} \
-        python3-indy-crypto=${python3_indy_crypto_ver} \
-        libindy-crypto=${indy_crypto_ver} \
-        vim
+	libsodium18 \
+	libbz2-dev \
+	zlib1g-dev \
+	liblz4-dev \
+	libsnappy-dev \
+	rocksdb=5.8.8 \
+	libindy \
+	ursa \
+	vim
+
+RUN pip3 install \
```

Comment:
what is the reason to switch installation to pip?
@skhoroshavin @Toktar can you recommend such way to install indy-node or the deb packages is preferable

Reply:
@andrewwhitehead

Reply:
I think pip is preferable. Python 3.5 is end-of-life, and I don't know what those deb packages look like inside but this should continue to work if a newer python is substituted.

Reply:
In fact, deb packages contains scripts that guarantees the correct environment and correct dependencies. Using pip for installing `indy-plenum` and `indy-node` is not recommended. Official deb packages SHOULD be used.

Reply:
If plenum and node depend on such specific dependencies to work then why are they published to Pypi at all?